kong/plugins/proxy-cache/handler.lua

```diff
@@ -356,8 +360,15 @@ function ProxyCacheHandler:access(conf)
     end
   end
 
-  res.headers["Age"] = floor(time() - res.timestamp)
-  res.headers["X-Cache-Status"] = "Hit"
+
+  if ngx.var.http_kong_debug then
```

Comment:
Not a blocker, but you could use `set_header` here, right?

Reply:
It was not clear why it was not the case before, so I have prefered to keep it as it. I will rebase on 2.7.0 and do some tests to check

Reply:
I get it, on the function `resè is retrieved from the cache
`  local res, err = strategy:fetch(cache_key)`
`res` is updated and answered with `  return kong.response.exit(res.status, res.body, res.headers)`
We have to set res.hearders we cannot use the helper here.